src/testrender/shading.cpp

```diff
@@ -30,157 +30,217 @@ enum ClosureIDs {
 
 // these structures hold the parameters of each closure type
 // they will be contained inside ClosureComponent
-struct EmptyParams      { };
-struct DiffuseParams    { Vec3 N; };
-struct OrenNayarParams  { Vec3 N; float sigma; };
-struct PhongParams      { Vec3 N; float exponent; };
-struct WardParams       { Vec3 N, T; float ax, ay; };
-struct ReflectionParams { Vec3 N; float eta; };
-struct RefractionParams { Vec3 N; float eta; };
-struct MicrofacetParams { ustring dist; Vec3 N, U; float xalpha, yalpha, eta; int refract; };
+struct EmptyParams {
+};
+struct DiffuseParams {
+    Vec3 N;
+};
+struct OrenNayarParams {
+    Vec3 N;
+    float sigma;
+};
+struct PhongParams {
+    Vec3 N;
+    float exponent;
+};
+struct WardParams {
+    Vec3 N, T;
+    float ax, ay;
+};
+struct ReflectionParams {
+    Vec3 N;
+    float eta;
+};
+struct RefractionParams {
+    Vec3 N;
+    float eta;
+};
+struct MicrofacetParams {
+    ustring dist;
+    Vec3 N, U;
+    float xalpha, yalpha, eta;
+    int refract;
+};
 
```

Comment:
This was more readable prior to clang-format.

Reply:
This is all about to be rewritten anyway